Add titles-only RSS feed

kevinlitchfield opened the pull request:
Adds a titles-only RSS feed accessible at `/rss-titles` to complement the full RSS feed available at `/rss`.

Sidebar now looks like this:

<img width="234" alt="screen shot 2017-03-24 at 1 46 37 pm" src="https://cloud.githubusercontent.com/assets/11491159/24306897/8fe621cc-1098-11e7-8281-262f7e2a7794.png">


Addresses #45.

rrrene commented on web/controllers/feed_controller.ex:
```diff
@@ -11,8 +11,13 @@ defmodule ElixirStatus.FeedController do
   end
 
-  def rss(conn, _params) do
+  def full_feed(conn, _params) do
```
Please revert all changes to this function.

rrrene commented on web/templates/shared/sidebar.html.eex:
```diff
@@ -23,7 +23,8 @@
       <a class="sidebar-nav-item no-hover-decoration" href="http://elixirweekly.net/" target="_blank"><span class="badge badge--new">NEW!</span> <span class="hover-decoration">ElixirWeekly</span></a>
       <a class="sidebar-nav-item" href="https://twitter.com/elixirstatus" target="_blank">Twitter</a>
-      <a class="sidebar-nav-item" href="/rss">RSS</a>
+      <a class="sidebar-nav-item" href="/rss">RSS (full)</a>
```
Please revert **all** changes to this file.

rrrene commented on web/controllers/feed_controller.ex:
```diff
@@ -11,8 +11,13 @@ defmodule ElixirStatus.FeedController do
+  def full_feed(conn, _params) do
     postings = Posting.published
-    render(conn, "rss.xml", postings: postings.entries)
+    render(conn, "full.xml", postings: postings.entries)
```
Please name all templates like their functions.

rrrene commented on web/controllers/feed_controller.ex:
```diff
@@ -11,8 +11,13 @@ defmodule ElixirStatus.FeedController do
+    render(conn, "full.xml", postings: postings.entries)
+  end
+
+  def titles_feed(conn, _params) do
```
Please rename `titles_feed` to `rss_titles_only`.

rrrene commented on web/router.ex:
```diff
@@ -39,7 +39,9 @@ defmodule ElixirStatus.Router do
 
-    get "/rss", FeedController, :rss
+    get "/rss", FeedController, :full_feed
+    get "/rss-titles", FeedController, :titles_feed
```
Please rename `rss-titles` to `rss-titles-only`

rrrene commented on web/controllers/feed_controller.ex:
```diff
@@ -11,8 +11,13 @@ defmodule ElixirStatus.FeedController do
+
+  def titles_feed(conn, _params) do
+    postings = Posting.published
+    render(conn, "titles.xml", postings: postings.entries)
```
Please rename `rss_titles_only.xml`.

kevinlitchfield commented:
@rrrene Thanks for the review, and apologies for the delay. Have addressed all feedback.

Since the titles-only RSS feed is no longer linked anywhere, it's not discoverable by browsing the site. Should it be linked somewhere or no?
